backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -279,6 +285,14 @@ def update(
 
             instance.add_items(zaken)
 
+        if add_data:
+            zaken = self._get_zaken(zaak_filters, add_data or [], bulk_select)
+            self.instance.add_items(zaken)
+
+        if remove_data:
+            zaken = self._get_zaken(zaak_filters, remove_data or [], bulk_select)
```

Comment:
```suggestion
            zaken = self._get_zaken(zaak_filters, remove_data, bulk_select)
```

Note: if `bulk_select` is true, this filter will **not** include zaken that are in a destruction list (and thus also not in this list). So I think that this will not work as intended :thinking:

Reply:
Correct, an oversight, the correct implementation depend on how we want to address: https://github.com/maykinmedia/open-archiefbeheer/pull/428#discussion_r1804426456